src/reducers/manageBand.js

```diff
@@ -3,6 +3,10 @@ export default function manageBand(state = {
 }, action) {
   switch (action.type) {
     case 'ADD_BAND':
+      /**
+       * Strictly speaking `Math.random()` doesn't prevent IDs from being
+       * the same, despite the odds being astronomically low.
+       */
```

Comment:
And here I'm just being exceptionally pedantic. 😓

Reply:
I tried using uuid, but couldn't get it to work. any thoughts?

Reply:
`uuid` is probably the right way to go. Stand by.